Src/Persistence/Repositories/Repository.cs

```diff
@@ -34,6 +34,18 @@ public IQueryable<TEntity> Find(Expression<Func<TEntity, bool>> predicate)
     {
         return _context.Set<TEntity>().Where(predicate);
     }
+
+    public async Task<bool> ExistsAsync(object id, CancellationToken cancellationToken)
```

Comment:
We can remove this and use AnyAsync, it is not good because it may traverse all data on DB and then return full data rather than make the comparison, it is better to use any method
@FarzanehMG